LayoutTests/fast/dom/gc-dom-tree-lifetime.html

```diff
@@ -71,16 +76,34 @@
     divY = null;
     divZ = null;
     gc();
+    numNodesDeleted += (prevNodes - window.internals.numberOfLiveNodes());
+});
+if (window.internals)
+    numNodesAfterTest = window.internals.numberOfLiveNodes();
+// DOM nodes can be destructed asynchronously during idle time. Check number of nodes destructed on idle callback.
+// Use the timeout to ensure idle time occurs on the event loop after the test logic finishes so that DOM nodes will 
+// be opportunistically destructed before doing the check.
+setTimeout(() => window.requestIdleCallback(checkNumberOfNodesAlive), 500);
```

Comment:
500 is an awfully long wait. Can we use a shorter timeout?
Maybe we need a way to manually trigger opportunistic tasks?
e.g. `internals.executeOpportunisticallyScheduledTasks`, which executes all opportunistic tasks.
We can fallback to `setTimeout` & `requestIdleCallback` when internals is not available.
I'm afraid the current approach will make these tests flaky.